modules/core/entity/farm_entity.post_update.php

```diff
@@ -24,3 +24,12 @@ function farm_entity_post_update_enforce_plan_eri(&$sandbox) {
 function farm_entity_post_update_rebuild_bundle_field_maps(&$sandbox = NULL) {
   \Drupal::service('entity_field.manager')->rebuildBundleFieldMap();
 }
+
+/**
+ * Uninstall EXIF Orientation module.
+ */
+function farm_entity_post_update_uninstall_exif_orientation() {
+  if (\Drupal::service('module_handler')->moduleExists('exif_orientation')) {
+    \Drupal::service('module_installer')->uninstall(['exif_orientation']);
```

Comment:
What happens if somebody copied the `farm_entity.info.yml` file's dependencies into a contrib module and inadvertently (or intentionally) depends on the `exif_orientation` module? Would this force their module to be uninstalled also, or would this fail?

Reply:
Yes that's true. We got burned by that issue ourselves when a contrib module uninstalled a submodule we were depending on.

I'll look into checking whether or not other modules depend on `exif_orientation` before uninstalling it.

The catch is: since we're adding the `exif` PHP extension, if the `exif_orientation` module stays installed then it will start working (it wasn't because we didn't have `exif`). And that means that original images will start to get rotated, losing their EXIF data in the process. But if someone is depending on `exif_orientation`, then I suppose that's on them...

Reply:
fixup: https://github.com/farmOS/farmOS/pull/941/commits/3d03f2d525bb722d0373ae249dc1d37d735ae8ad